android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerModule.java

```diff
@@ -220,6 +220,15 @@ public void configure(LongPressGestureHandler handler, ReadableMap config) {
         handler.setMaxDist(PixelUtil.toPixelFromDIP(config.getDouble(KEY_LONG_PRESS_MAX_DIST)));
       }
     }
+
+    @Override
+    public void extractEventData(LongPressGestureHandler handler, WritableMap eventData) {
+      super.extractEventData(handler, eventData);
+      eventData.putDouble("x", PixelUtil.toDIPFromPixel(handler.getLastRelativePositionX()));
```

Comment:
Should we add it to base class `HandlerFactory` implementation instead of having it here maybe? How do we handle that on iOS?

Reply:
Please see at description, @kmagiera:
"However, we do not provide support for Rotation, because it has no sense and is not done on iOS. So I decided no to export these fields to JS in case of rotation GH"